channeldb/payments.go

```diff
@@ -361,7 +361,7 @@ func deserializeHop(r io.Reader) (*route.Hop, error) {
 
 func serializeRoute(w io.Writer, r route.Route) error {
 	if err := WriteElements(w,
-		r.TotalTimeLock, r.TotalFees, r.TotalAmount, r.SourcePubKey[:],
+		r.TotalTimeLock, r.TotalAmount, r.SourcePubKey[:],
```

Comment:
note: this is a safe change since the code is not yet used anywhere.

Reply:
Phew, was wondering why we weren't handling that 😅